src/TLuaInterpreterMapper.cpp

```diff
@@ -2520,6 +2529,7 @@ int TLuaInterpreter::lockSpecialExit(lua_State* L)
 
     lua_pushboolean(L, true);
     host.mpMap->setUnsaved(__func__);
+    host.mpMap->update();
```

Comment:
Locking an exit does not produce any visible changes - so this is not needed.

Reply:
See https://github.com/Mudlet/Mudlet/pull/7187#discussion_r1544529665